examples/codes/run_plasmod_example.py

```diff
@@ -249,3 +249,16 @@ def plot_profile(plasmod_solver, var_name, var_unit):
 plasmod_solver.problem_settings["qdivt_sup"] = 10.0
 plasmod_solver.run()
 print_outputs(plasmod_solver)
+
+# %%[markdown]
+# #### Changing the mapping sending or recieving
+# The mapping can be changed on a given parameter or set of parameters.
+# Notice how the value of `q_95` doesn't change in the output
+# even though its value has in the parameter (the previous value of 3.5 is used).
+
+# %%
+plasmod_solver.modify_mappings({"q_95": {"send": False}})
+plasmod_solver.params.q_95 = (5, "input")
+plasmod_solver.run()
+print_outputs(plasmod_solver)
+print("\nq_95 value history\n", plasmod_solver.params.q_95.history())
```

Comment:
I think it is fine if:
* when we modify a mapping to no longer send, the last value is kept by default
* we can then set a value on a parameter as shown above to overwrite the last value (if necessary)

Reply:
Yeah, for plasmod definitely that is what happens